readme.md

```diff
@@ -616,6 +615,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [cool-ascii-faces](https://github.com/maxogden/cool-ascii-faces) - Get some cool ascii faces.
 - [cat-ascii-faces](https://github.com/melaniecebula/cat-ascii-faces) - ₍˄·͈༝·͈˄₎◞ ̑̑ෆ⃛ (=ↀωↀ=)✧ (^･o･^)ﾉ”
 
+### Web Server
+
+- [http-server](https://github.com/nodeapps/http-server) - Simple, zero-config command-line HTTP server.
+- [Harp](https://github.com/cheeriojs/cheerio) - Static web server with built-in preprocessing like Markdown, Jade, EJS, LESS, Stylus, Sass and CoffeeScript.
```

Comment:
`Static web server with built-in preprocessing.` or reduce the words after `like` to 2 or 3.

And I doubt whether we can regard a server including pre-processing function as a "static server", though I don't understand the term "static server" well.